test/acceptance/features/steps/cluster.py

```diff
@@ -177,7 +179,17 @@ def create_identity(self, namespace: str, service_account: str, secret: str, tim
         corev1 = client.CoreV1Api(api_client)
 
         body = client.V1ServiceAccount(metadata=client.V1ObjectMeta(name=service_account))
-        sa = corev1.create_namespaced_service_account(namespace=namespace, body=body)
+        sa_list = corev1.list_namespaced_service_account(
+            namespace=namespace,
+            field_selector=f'metadata.name={service_account}')  # type: client.V1ServiceAccountList
+        if len(sa_list.items) != 0:
+            # the service account already exists, so update it instead of creating
+
+            # FIXME: if we end up implementing multi-threaded testing, this is
+            # going to break a lot of things very fast.
+            sa = corev1.replace_namespaced_service_account(service_account, namespace, body)
+        else:
+            sa = corev1.create_namespaced_service_account(namespace=namespace, body=body)
```

Comment:
If we configure a new tenant for each scenario it should never happen right? Are we creating a new tenant?

Reply:
do you think adding the scenario_id as a suffix to the tenant name may be easy to do? this should for naming conflict here

Reply:
BTW, I'd say this may be done in a new PR focused on parallel testing

Reply:
Yeah, I'm happy to punt this investigation to a separate PR.  The intent with this PR was to get it working for provided clusters, not with parallel testing (that's going to be a bit more work).

Reply:
great, can you create a story for this?